Gemfile

```diff
@@ -2,18 +2,14 @@ source 'https://rubygems.org'
 ruby "2.4.2"
 
 gem 'rails', '4.2.10'
-gem 'rails_12factor', group: :production
 
 gem 'asciidoctor', '>=1.5.4'
```

Comment:
The deploy of the review app failed. From the logs it seems the uglify version can't parse our assets. This happened before, and the fix was to lock the uglify version. At some time I was trying to debug the problem and was able to deploy without the locked version. It seems it was an error. 

Summary: can you basically revert this commit https://github.com/git/git-scm.com/commit/a8359d5abba0d90959ff173814c354918cb2fa08 ? It should make the app deploy work.